examples/fnn/mnist_conv_fnn.ml

```diff
@@ -5,8 +5,8 @@ open Tensorflow_fnn
 
 let label_count = Mnist_helper.label_count
 let image_dim = Mnist_helper.image_dim
-let epochs = 2000
 let batch_size = 256
+let epochs = 235
```

Comment:
Thanks! Could you do the same for mnist_linear_fnn.ml and mnist_multi_fnn.ml in the same directory ? Besides that everything else looks good.

Reply:
there are no changes to the `mnist_linear_fnn.ml` nor `mnist_multi_fnn.ml`, as the fit function in these examples are not supplied the `~batch_size argument` hence the behavior does not change (as batch_size is equal to the input size). 

The `batch size` in these examples are only used when testing the model, and is passed to `Mnist_helper.batch_accuracy`

Reply:
Makes sense, thanks.